app/memorial/routes.py

```diff
@@ -83,3 +83,12 @@ def settings(memorial_id):
         return {"msg": response}, code
 
     return {"memorial": response}, 200
+
+
+@memorial_bp.route('/members/<memorial_id>', methods=['GET'])
+@jwt_required(optional=True)
```

Comment:
this shouldn't be optional here
```suggestion
@jwt_required()
```